### PR TITLE
ISSD-2427 [Testray] Sort Factors and Options alphabetically

### PR DESCRIPTION
**Pull request by abfranca**

### CODE REVIEW

**Comment by abfranca** [comment]
ci:test:sf

**Review by nNilton — APPROVED** [review]

File: workspaces/liferay-testray-workspace/client-extensions/liferay-testray-custom-element/src/services/rest/TestrayFactorCategory.ts [hunk 1]
```diff
@@ -40,7 +40,9 @@ class TestrayFactorCategoryRest extends Rest<
 			);
 
 			if (response?.items) {
-				factorCategoryItems.push(response.items);
+				factorCategoryItems.push(
+					response.items.sort((a, b) => a.name.localeCompare(b.name))
```
Comment (claricebrasil) [inline workspaces/liferay-testray-workspace/client-extensions/liferay-testray-custom-element/src/services/rest/TestrayFactorCategory.ts:44]:
```suggestion
					[...response.items].sort((a, b) => a.name.localeCompare(b.name))
```

**Comment by claricebrasil** [comment]
ci:test:sf

**Comment by abfranca** [comment]
ci:test:sf

**Comment by abfranca** [comment]
ci:test:sf

**Comment by claricebrasil** [comment]
resent: #5419

**Pull request closed without merging**